tests/set-labels/both-additionalLabelFields-fieldSpecs/.expected/results.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1alpha2
+kind: FunctionResultList
+metadata:
+  name: fnresults
+exitCode: 1
+items:
+  - image: gcr.io/kpt-fn/set-labels:unstable
+    stderr: '[error] /// : failed to configure function: fieldSpecs has been deprecated, please rename it to additionalLabelFields'
```

Comment:
K6: https://github.com/GoogleContainerTools/kpt/blob/main/docs/style-guides/errors.md

Honestly at this point, the expectation is the style guides are followed without being asked every time. There are a handful of rules.

Reply:
Thanks for pointing this out.
I usually pay more attention to markdown file and sometimes forget to check the styling in the code (error message). I will start to pay more attention to it too.